data/json/items/gun/45.json

```diff
@@ -92,6 +92,18 @@
     "magazine_well": 1,
     "magazines": [ [ "45", [ "m1911mag", "m1911bigmag" ] ] ]
   },
+  {
+    "id": "m1911_MEU",
+    "copy-from": "m1911",
+    "type": "GUN",
+    "name": "M45A1",
+    "description": "The M45A1 supplanted earlier M45 MEUSOC pistols in use by Force Recon elements of Marine Expeditionary Units.  Where the original M45 pistols were gutted M1911A1's hand-fitted by USMC armorers, the updated M45A1's are hardly different from any other commercial 1911 design save for their dual recoil springs.  Most were replaced in 2016 with Glock 19's due to logistics issues.",
```

Comment:
Please remove any exact date from description.

Reply:
I think we're fine for dates of things that have actually happened. It's not a speculative date for something that might be affected by cata lore.